src/api/ledger/accountinfo.js

```diff
@@ -18,21 +55,23 @@ function formatAccountInfo(response) {
   });
 }
 
-function getAccountInfoAsync(account, options, callback) {
-  validate.address(account);
-  validate.getAccountInfoOptions(options);
+function getAccountInfoAsync(account: string, options: AccountInfoOptions,
+  callback: AccountInfoCallback) {
+    validate.address(account);
+    validate.getAccountInfoOptions(options);
 
-  const request = {
-    account: account,
-    ledger: options.ledgerVersion || 'validated'
-  };
+    const request = {
```

Comment:
Here is the eslint issue. Looks like someone else had the same idea: https://github.com/eslint/eslint/issues/3173